lms/envs/production.py

```diff
@@ -1128,3 +1128,11 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+# Figures configuration
+if 'figures.apps.FiguresConfig' in INSTALLED_APPS:
+    from figures.settings.lms_production import *
+    ENV_TOKENS.setdefault('FIGURES', {})
+    update_webpack_loader(WEBPACK_LOADER, ENV_TOKENS['FIGURES'])
```

Comment:
Why we need to update webpack loader? we are not suppose to use figure frontend.

Reply:
removed